harvard-university-for-the-creative-arts.csl

```diff
@@ -110,11 +125,11 @@
           </else-if>
         </choose>
       </if>
-      <else-if type="manuscript song post-weblog post personal_communication speech" match="any">
-        <group delimiter=". ">
+      <else-if type="manuscript song post-weblog post" match="any">
+        <group delimiter=" ">
           <text variable="genre" prefix="[" suffix="]"/>
           <text variable="medium" prefix="[" suffix="]"/>
-          <text variable="note" prefix="[" suffix="]"/>
+          <text variable="note" suffix="."/>
```

Comment:
remind me why we're using note here? I'm trying hard to get rid of that in as many styles as possible, given that it produces really weird output for many users.